plane/source/docs/guided-mode.rst

```diff
@@ -4,10 +4,33 @@
 GUIDED Mode
 ===========
 
-The GUIDED mode is used when you want the aircraft to fly to a specific
-point on the map without setting up a mission. Most ground control
+GUIDED mode is used when you want the aircraft to fly to a specific
+point on the map, or in specific direction (heading) without setting up a mission. 
+
+Location sub-mode
+-----------------
+Most ground control
 stations support a "click to fly to" feature where you can click a point
-on the map and the aircraft will fly to that location then loiter.
+on the map and the aircraft will fly to that location then loiter. If an 
+altitude is requested, the aircraft will fly at the current altitude until
+it reaches the target location, then will loiter down or up to the requested
+altitude. 
+
+Heading sub-mode
+----------------
+The guided mode heading feature isn't supported by Mission Planner or QGroundControl, but can be
+requested via MavLink commands or using Lua. If the aircraft receives a GUIDED_CHANGE_HEADING
+command it will fly indefinitely towards the heading in degrees. 
+
+The target altitude and desired airspeed or
+groundspeed of the vehicle to fly at can be set using the GUIDED_CHANGE_ALTITUDE and GUIDED_CHANGE_SPEED
+MavLink commands. If the aircraft is flying in Guided "heading" sub-mode then it will try to
+achieve the altitude as it flies, as there is no "end point" to a heading target for a final loiter.
+
+If the vehicle is flying in Guided "Location sub-mode" then GUIDED_CHANGE_ALTITUDE and GUIDED_CHANGE_SPEED 
```

Comment:
move to the location sub-mode section....people have a tendency to read only what they are interested in